aws/table_aws_wellarchitected_lens.go

```diff
@@ -0,0 +1,240 @@
+			},
+			{
+				Name:        "updated_at",
+				Description: "The date and time when the lens was last updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "The description of the lens.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lens_status",
+				Description: "The status of the lens.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lens_type",
+				Description: "The type of the lens.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lens_version",
+				Description: "The version of the lens.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "owner",
+				Description: "An Amazon Web Services account ID.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "share_invitation_id",
+				Description: "The ID assigned to the shared invitation.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedLens,
+				Transform:   transform.FromField("ShareInvitationId"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LensName"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWellArchitectedLens,
+				Transform:   transform.FromField("Tags"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("LensArn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listWellArchitectedLenses(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := WellArchitectedClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_wellarchitected_lens.listWellArchitectedLenses", "connection_error", err)
+		return nil, err
+	}
+	if svc == nil {
+		// Unsupported region, return no data
+		return nil, nil
+	}
+
+	// Limiting the results
+	maxLimit := int32(50)
+	if d.QueryContext.Limit != nil {
+		limit := int32(*d.QueryContext.Limit)
+		if limit < maxLimit {
+			maxLimit = limit
+		}
+	}
+
+	input := &wellarchitected.ListLensesInput{
+		MaxResults: maxLimit,
+	}
+
+	equalQuals := d.EqualsQuals
+	if equalQuals["lens_status"] != nil {
+		if equalQuals["lens_status"].GetStringValue() != "" {
+			input.LensStatus = types.LensStatusType(equalQuals["lens_status"].GetStringValue())
+		}
+	}
+	if equalQuals["lens_name"] != nil {
+		if equalQuals["lens_name"].GetStringValue() != "" {
+			input.LensName = aws.String(equalQuals["lens_name"].GetStringValue())
+		}
+	}
+	if equalQuals["lens_type"] != nil {
+		if equalQuals["lens_type"].GetStringValue() != "" {
```

Comment:
can we use `EqualsQualString` instead of `EqualsQuals`?

Reply:
Updated the code to use` EqualsQualString`